jquery.dynatable.js

```diff
@@ -315,6 +315,11 @@
 
       td += '"';
     }
+    
+    // allow cells to be editable
+    if (column.contentEditable) {
+      td += ' contenteditable="true"';
```

Comment:
For the attribute name, I was referring to the fact that you have `contenteditable` instead of `contentEditable` (capitalized "Editable").

Yeah, I'm thinking as a data attribute, like `data-contenteditable` or maybe namespaced to avoid conflicts with other libraries like `data-dynatable-contenteditable` on the `th` element in the column header cell.